content/intro-to-storybook/react/ko/get-started.md

```diff
@@ -1,77 +1,79 @@
+
+   
 ---
-title: 'React를 위한 Storybook 튜토리얼'
+title: '리액트(React)를 위한 스토리북(Storybook) 튜토리얼'
 tocTitle: '시작하기'
-description: 'Storybook을 개발 환경에 설치해봅시다'
-commit: 'b66f341'
+description: '스토리북(Storybook)을 개발 환경에 설치해보세요'
 ---
 
-Storybook은 개발 모드에서 앱과 함께 실행됩니다. 이것은 비즈니스 로직과 컨텍스트로부터 UI 컴포넌트를 독립적으로 분리하여 만들수 있도록 도와줍니다. 본 Storybook 문서는 React를 위한 것입니다. 그 밖의 [React Native](/intro-to-storybook/react-native/en/get-started), [Vue](/intro-to-storybook/vue/en/get-started), [Angular](/intro-to-storybook/angular/en/get-started) 그리고 [Svelte](/intro-to-storybook/svelte/en/get-started)에 대한 문서도 있습니다.
+스토리북(Storybook)은 개발 모드에서 앱과 함께 실행됩니다. 스토리북은 비즈니스 로직과 맥락(context)으로부터 분리된 UI 컴포넌트를 만들 수 있도록 도와줍니다. 현재 문서는 리액트(React)를 위해 작성되었습니다. 그 밖의 [React Native](/intro-to-storybook/react-native/en/get-started), [Vue](/intro-to-storybook/vue/en/get-started), [Angular](/intro-to-storybook/angular/en/get-started) [Svelte](/intro-to-storybook/svelte/en/get-started) 및 [Ember](/intro-to-storybook/ember/en/get-started)에 대한 다른 문서도 존재합니다.
+
+![스토리북과 여러분의 앱](https://storybook.js.org/tutorials/intro-to-storybook/storybook-relationship.jpg)
 
-![Storybook과 여러분의 앱](/intro-to-storybook/storybook-relationship.jpg)
+## 리액트 스토리북 설정
 
-## React Storybook 설치하기
+우리의 개발 환경에 빌드(build) 프로세스를 설정하려면 몇 가지 단계를 따라야 합니다. 우선 우리는 [degit](https://github.com/Rich-Harris/degit)을 사용하여 빌드 시스템을 설정하려고 합니다. 이 패키지를 사용해 "템플릿"(일부 기본 구성으로 부분적으로 구축된 애플리케이션)을 다운로드해 개발 작업 흐름(workflow)을 빠르게 파악할 수 있습니다.
 
-개발 환경에 빌드 프로세스를 설정하려면 몇 가지 단계를 거쳐야 합니다. 먼저 [React App 생성하기](https://github.com/facebook/create-react-app)(CRA)를 사용하여 빌드 시스템을 설정하고 [Storybook](https://storybook.js.org/)과 [Jest](https://facebook.github.io/jest/) 테스트를 앱에서 활성화해야 합니다. 아래의 명령어들을 실행해주세요:
+다음 명령을 실행해주세요:
 
 ```bash
-# Create our application:
-npx create-react-app taskbox
+# 템플릿 복제
+npx degit chromaui/intro-storybook-react-template taskbox
 
 cd taskbox
 
-# Add Storybook:
-npx -p @storybook/cli sb init
+# 의존성(dependency) 설치
+yarn
 ```
```

Comment:
@lisohh if you or your fellow collaborators could revert the comments in the snippets in this pull request back to English, I'd appreciate it. I'm aware that translating them brings it "closer to home" (pardon the bad pun) but we'd prefer to leave them until we have a better solution for them.